modules/programs/neovim.nix

```diff
@@ -232,6 +248,20 @@ in {
       coc = {
         enable = mkEnableOption "Coc";
 
+        plugins = mkOption {
+          type = with types; listOf (either package pluginWithConfigType);
+          default = [ ];
```

Comment:
from a backcompat perspective, it'd be a little ugly but this could be filled in with `filter (p: startsWith "coc-" p.name) cfg.plugins` by default to fix existing user configs